test/extensions/filters/http/aws_request_signing/config_test.cc

```diff
@@ -51,6 +49,29 @@ region: us-west-2
   cb(filter_callbacks);
 }
 
+TEST(AwsRequestSigningFilterConfigTest, RouteSpecificFilterConfig) {
+  const std::string yaml = R"EOF(
+aws_request_signing:
```

Comment:
Do we want to include `host_rewrite` for line coverage?

Reply:
yea good idea.

Reply:
Updated